kubernetes_platform/proto/kubernetes_executor_config.proto

```diff
@@ -156,6 +157,23 @@ message ConfigMapAsEnv {
     repeated ConfigMapKeyToEnvMap key_to_env = 2;
 }
 
+message GenericEphemeralVolume {
+    // more details in https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#generic-ephemeral-volumes
+    // Name of the ephemeral volume.
+    string volume_name = 1;
+    // Container path to mount the volume
+    string mount_path = 2;
+    // Corresponds to ephemeral.volumeClaimTemplate.spec.accessModes field.
+    repeated string access_modes = 3;
+    // Corresponds to ephemeral.volumeClaimTemplate.spec.resources.requests.storage field.
+    string size = 4;
+    // If true, corresponds to omitted ephemeral.volumeClaimTemplate.spec.storageClassName.
+    bool default_storage_class = 5;
+    // Corresponds to ephemeral.volumeClaimTemplate.spec.storageClassName string field.
+    // Should only be used if default_storage_class is false.
+    string storage_class_name = 6;
+}

```

Comment:
According to the documentations, ephemeral volume also support metadata and resources (requests, limits, claims). Would you mind adding those spec as well? Thank you.

You can reuse the proto message `PodMetadata` for metadata.

Reply:
Wow, that was a super fast review

For resources: I don't think that they support resources other that "requests.storage", which is what the "size" is there (it's a tiny abstraction of the k8s terminology, but i don't find it that confusing).

For metadata: sure, will add that asap.